src/offset/local/tz_info/timezone.rs

```diff
@@ -52,6 +52,14 @@ impl TimeZone {
             return Self::from_file(&mut file);
         }
 
+        // attributes are not allowed on if blocks in Rust 1.38
+        #[cfg(target_os = "android")]
```

Comment:
should this moved up to above line 46? Also what do we do in the case that the `tz_string` is "localtime" on android?

Reply:
I  have yet to find exhaustive documentation on this, but from what I can tell, `tzdata` doesn't contain a `localtime` entry, and there is `/etc/localtime` like on other Unix systems. Moving the `tzdata` block up sounds good to me.

Reply:
I've checked an Android device here, and it does not have an /etc/localtime - perhaps they did away with it in later Android releases, or it varies by manufacturer? Android appears to set TZ to the timezone configured in the settings by default, so presumably users would not hit this unless they explicitly overwrote the default value.